catalog/ui/catalog-ui-search/src/main/webapp/component/search-form/search-form-collection.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ /*global require*/
+ var _ = require('underscore');
+ var Backbone = require('backbone');
+ var SearchForm = require('./search-form');
+
+ module.exports = Backbone.Collection.extend({
+   model: SearchForm,
+   initialize: function(models, options) {
+       this.add(new SearchForm({type: "basic", queryModel: options[0].queryModel}));
+       this.add(new SearchForm({type: "text", queryModel: options[0].queryModel}));
+   },
+   getCustomQueryForms: function() {
```

Comment:
Is this for a later PR to utilize?

Reply:
yes, there will be a follow-on PR that will utilize this